modules/gathering-data-specific-features.adoc

```diff
@@ -56,7 +56,14 @@ ifndef::openshift-rosa,openshift-dedicated[]
 |Data collection for {sandboxed-containers-first}.
 
 |`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed-version-NHC>`
-|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO) Operator.
+|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).
+
+Use this image if your NHC Operator version is less than 0.9.0. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.
```

Comment:
Looks much better. NIT: Would it be better to highlight the difference between the two images using bold?
```suggestion
Use this image if your NHC Operator version is **less than 0.9.0**. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.
```